cli/src/main.rs

```diff
@@ -52,10 +70,19 @@ async fn main() -> Result<(), color_eyre::Report> {
     for arg in env::args().skip(1) {
         if !arg.is_empty()
             && !([HELP_ARG, SUBMIT_ARG]
-                .iter()
-                .any(|group| group.contains(&arg.as_str())))
+                 .iter()
+                 .any(|group| group.contains(&arg.as_str())))
         {
             print_help(&styling)?;
+			// NOTE: because of how `color_eyre` works, we need to
+			// install the hook before creating any instance of
+			// `eyre::Report`, otherwise the `eyre` default reporting
+			// hook is going to be installed automatically.
+
+			// This results in a nasty repetition of the
+			// `color_eyre::install().unwrap()` pattern, which is the
+			// lesser of two evils
+			color_eyre::install().expect("CRITICAL");
```

Comment:
I assume this is related to #2212

Reply:
Also, I see this hook install around the codebase in different crates. If it's expected to fail, you think we could make it into a function?

Reply:
It's not expected to fail, the point of this particular implementation is that we know for sure that we haven't installed the hook. And we have no reason to expect that the user would want to disable it (vic config params, because they're unreadable). Afterwards, it's a different story.